src/league-of-legends-api/poll-current-game.ts

```diff
@@ -13,31 +13,85 @@ const httpsAgent = new https.Agent({
   cert: fs.readFileSync('src/league-of-legends-api/riotgames.pem')
 });
 
-let cachedGame: RootGameObject;
+let cachedEvents: Event[] = [];
```

Comment:
Where is this actually used?

Reply:
It's used now whoops I was moving too fast